packages/vscode-extension/src/handlers.ts

```diff
@@ -420,13 +420,22 @@ export async function addResourceHandler(args?: any[]): Promise<Result<null, FxE
     namespace: "fx-solution-azure",
     method: "addResource",
   };
-  const localEnvManager = new LocalEnvManager(VsCodeLogInstance, ExtTelemetry.reporter, VS_CODE_UI);
-  const projectSettings = await localEnvManager.getProjectSettings(ext.workspaceUri.fsPath);
-  const includeBackend = ProjectSettingsHelper.includeBackend(projectSettings);
+  let excludeBackend = true;
+  try {
+    const localEnvManager = new LocalEnvManager(
+      VsCodeLogInstance,
+      ExtTelemetry.reporter,
+      VS_CODE_UI
+    );
+    const projectSettings = await localEnvManager.getProjectSettings(ext.workspaceUri.fsPath);
+    excludeBackend = ProjectSettingsHelper.includeBackend(projectSettings);
```

Comment:
The naming is confusing, does it mean include or exclude?

Reply:
When current project includes backend, then auto npm install for backend will be excluded because it has been done before. So I think using `excludeBackend` is better.

Reply:
It's clear to me once you explained. I just worry that others might get confused when they read code. Anyway, it's not a big problem